src/index.js

```diff
@@ -13,7 +13,10 @@ const log = v => {
   console.log(`Solved in ${end - start}ms`);
 };
 const readFile = promisify(fs.readFile);
-const getLines = c(init, split("\n"));
+const getLines = c(
+  ifElse(x => x.length > 1, init, head),
```

Comment:
Careful - I think this `head` will pass to "inputLines" parameter a string instead of an array of strings.

If that's what you wanted I'll slowly walk away.... 🙈